"Bare-metal mode" throws AWS init error

# Summary
When launching the application locally, it throws the following stack trace:
```
Caused by: java.lang.IllegalArgumentException: Could not resolve placeholder 'cloud.aws.credentials.access-key' in value "${cloud.aws.credentials.access-key}"
...
```
Loading of the AWS helper stack is disabled in the bare-metal config but seems to still be activated. Need to figure out a way to either work around it, or to trick AWS into loading silently because we don't use AWS stuff locally anyways.

# Implementation
#### Option A: Disable AWS helper stack
- [x] Find correct configuration values to disable the AWS helpers when running in the default profile (bare-metal mode) in `application.yml`.
#### Option B: Trick AWS initializer into loading
- [ ] Find way to send a dummy string as the access key and user ID so the library loads without throwing an exception.

 
*WARNING: Do not use a login string that has the same structure as a real key (i.e. starts with AKIA....). Even if the string is fake there are bots around GitHub that take actions on repositories with supposedly leaked keys. It's best that we don't come up in their searches, even with fake login creds.*

--- a/src/main/java/com/virtudoc/web/configuration/CloudConfiguration.java
+++ b/src/main/java/com/virtudoc/web/configuration/CloudConfiguration.java
@@ -15,6 +15,7 @@ import org.springframework.context.annotation.Primary;
 import org.springframework.context.annotation.Profile;
 
 @Configuration
+@Profile(value={"prod", "test", "dev-managed"})
 public class CloudConfiguration {
     @Value("${cloud.aws.credentials.access-key}")
     private String accessKey;

--- a/src/main/java/com/virtudoc/web/entity/Role.java
+++ b/src/main/java/com/virtudoc/web/entity/Role.java
@@ -13,7 +13,7 @@ public class Role {
 
     private String name;
 
-    @ManyToMany(mappedBy = "role")
+    @ManyToMany()
     private Set<UserAccount> users;
 
     public Long getId() {

--- a/src/main/resources/application.yml
+++ b/src/main/resources/application.yml
@@ -2,7 +2,7 @@ spring:
   application:
     name: web
   datasource:
-    url: jdbc:h2:mem:virtudoc
+    url: "jdbc:h2:mem:"
     username: h2user
     password: virtudoc
     driverClassName: org.h2.Driver

